doc/build-windows.md

```diff
@@ -67,6 +67,17 @@ build process.
 
 See also: [dependencies.md](dependencies.md).
 
+## Install NSIS
+
+The following steps are necessary to create the binary file. In order to create it, the system must
```

Comment:
s/binary file/binary installer

Reply:
I didn't understand your comment. Should I replace 'binary file' to 'binary installer', or the commands in the terminal should run in 's/binary file/binary installer/' directory?

Reply:
You should replace binary file with binary installer